apps/docs/src/app/core/component-docs/calendar/calendar-header/calendar-header.component.html

```diff
@@ -1,18 +1,39 @@
 <header>Calendar</header>
 <description>
-    Commonly used as the contents of a popover when composing “date-picker”, rarely used on its own as a standalone
-    component.
-    <br /><br />
-    The Time Picker component <strong>relies on provided datetime implementation</strong>
-    (<code>DatetimeAdapter</code>).
-    <br />
-    Currently fundamental-ngx offer 2 kind of adapters: 
-    FdDateTimeAdapter and <a fd-link routerLink="/core/moment-datetime-adapter">MomentDateTimeAdapter</a>.
-    <br /><br />
-    The calendar header buttons are navigable with the tab key. Pressing enter or the space bar when the month or year
-    is focused will change the day selection grid to the month or year selection grid respectively. The user can then
-    tab into the day/month/year selection grid and use the arrow keys to navigate around the grid and make a selection
-    with the space bar or enter key.
+    <p>        
+        Commonly used as the contents of a popover when composing “date-picker”, rarely used on its own as a standalone
+        component.
+        <br /><br />
+        The calendar header buttons are navigable with the tab key. Pressing enter or the space bar when the month or year
+        is focused will change the day selection grid to the month or year selection grid respectively. The user can then
+        tab into the day/month/year selection grid and use the arrow keys to navigate around the grid and make a selection
+        with the space bar or enter key.
+    </p>
+    <fd-message-strip [dismissible]="false" type="information">
+        <b>IMPORTANT</b>
+        <br>
+        The Time Picker component relies on provided datetime 
+        implementation (DatetimeAdapter) and datetime formats (DateTimeFormats).
+        <br>
+        The current examples are based on FdDateTimAdapter.
+        <br>
+        The FdDateTimAdapter is based on native date api (formatting is based on
+        <a href="https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat">
+            Intl object
+        </a>, parsing is based on
+        <a href="https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Date/parse">
```

Comment:
```suggestion
        <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/parse">
```
and same in other files

Reply:
changed! thanks a lot!